lib/msal-node/src/client/UsernamePasswordClient.ts

```diff
@@ -53,7 +53,7 @@ export class UsernamePasswordClient extends BaseClient {
         );
 
         const httpVerToken = response.headers?.[HeaderNames.X_MS_HTTP_VERSION];
-        atsMeasurement?.addStaticFields({
+        atsMeasurement?.add({
```

Comment:
Why does msal-node have a measurement at all? This can probably be removed?

Reply:
Good point, thank you. Removed it.